core/src/avm2/object.rs

```diff
@@ -1429,6 +1402,7 @@ pub enum WeakObject<'gc> {
     FontObject(FontObjectWeak<'gc>),
     LocalConnectionObject(LocalConnectionObjectWeak<'gc>),
     SharedObjectObject(SharedObjectObjectWeak<'gc>),
+    SoundTransformObject(SoundTransformObjectWeak<'gc>),
```

Comment:
Any strong reason for doing this? I think I'd prefer this to keep it on the AS3 side, especially if others like `Point` are also on the AS3 side.

Reply:
There aren't any accessible slots, unlike in Point, and we need to access the properties from `display_object.rs`, so I thought this worked better than the repeated `get_slot`s in there.